schema_based_on_diagram.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE "animals"(
```

Comment:
- [ ] According to the project requirements, we are required to  create a table between medical_histories and treatments as seen in the screenshot below
![Screenshot from 2022-10-25 15-48-21](https://user-images.githubusercontent.com/46388113/197821353-8b6038f4-9866-457f-b62b-49c0df346012.png)